lib/iris/fileformats/grib/message.py

```diff
@@ -33,7 +33,7 @@
 from iris.exceptions import TranslationError
 
 
-class _GribMessage(object):
+class GribMessage(object):
     """
     Lightweight GRIB message wrapper, containing **only** the coded keys and
     data attribute of the input GRIB message.
```

Comment:
This dosctring is basically a clone of the old 'GribWrapper' one, and wants revisiting.
I don't know if we still have an ambition to limit it to coded keys, but at present of course it does not so maybe best not even to say that.
Suggest something like.. """An object wrapping a grib_api message ID and exposing grib keys via the "sections" property."""